gen-resourcesdocs/pkg/outputs/kwebsite/files.go

```diff
@@ -63,7 +63,6 @@ func escapeName(parts ...string) string {
 
 // headingID returns the ID built by hugo for a given header
 func headingID(s string) string {
-	result := strings.ToLower(s)
-	result = strings.ReplaceAll(result, " ", "-")
+	result := strings.ReplaceAll(s, " ", "-")
```

Comment:
Is it necessary to normalize the heading?
I was under the impression that Hugo would do this.

Reply:
Yes, Hugo does it by default, but puts the anchor in lowercase. I would prefer to keep the original case, as these are Resource names, and I would like to use the anchor name as text to display in the api-reference shortcode: (see https://github.com/kubernetes/website/pull/23294/files#diff-4992971bcbda91c2f2374a2e62fffda3d7852174780e027070e766c62bf87ac1 )

Reply:
I need to look at the generated pages.
Could you update the shortcode to adapt the formatting of the resource name?

Reply:
The shortcode has been changed in this commit: https://github.com/kubernetes/website/pull/23294/commits/d9b18d378815988960f503e47ec8c0a8d32b3feb

The result should be visible in the paragraph https://deploy-preview-23294--kubernetes-io-master-staging.netlify.app/docs/concepts/configuration/secret/#using-imagepullsecrets  (see the PodSpec API ...)

The build of the preview website fails, I can't see why

Reply:
looking now. See this message:
This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

Reply:
You need to rebase?

Reply:
I think I made a rebase to restart the tests in the meantime. Here is the commit:
https://github.com/kubernetes/website/pull/23294/commits/99e1b4122221066583700d710228070988d75234

Reply:
I also get the message, but you can see the change in the api-reference.html file of this commit